app/src/main/java/org/jellyfin/androidtv/ui/shared/StrokeTextView.kt

```diff
@@ -0,0 +1,46 @@
+package org.jellyfin.androidtv.ui.shared
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.util.AttributeSet
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.core.content.ContextCompat
+import org.jellyfin.androidtv.R
+
+class StrokeTextView @JvmOverloads constructor(
+	context: Context,
+	attrs: AttributeSet? = null,
+) : AppCompatTextView(context, attrs) {
+	var strokeWidth = 0.0f
+	private var isDrawing: Boolean = false
+
+	init {
+		val styledAttrs = context.obtainStyledAttributes(attrs, R.styleable.StrokeTextView)
```

Comment:
From linter warning:

`This TypedArray should be recycled after use with #recycle()`